### DISCUSSION TITLE
Final Thinking Cap

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/22S-ASIAAM-191A#38] [Assignments]
> ## What are your thoughts about the future of web development for civil society? Which visiting voice left the most impact on you? What personal connections (if any) would you like to maintain from this class going forward?

## Submission
Respond to the above prompt in at least **4-5 sentences** before Wednesday, 6/1 and respond to at least one other student.

**Comment by daikis12** [comment]
As we've learned in this class throughout the quarter, with an increase in awareness of the intersection of ethics and technology and its impact on civil society, the future of web development for civil society should be more inclusive and community-oriented, instead of a handful of people dictating what/how to present. Particularly, the readings about Design/Data Justice reminded me of how even things like design and data are accessed inequitably within society, and so deconstructing the way our society is currently structured by giving historically marginalized people a technological platform to express themselves is a way to make society more equitable, which should be an ideal future of web development. Yoh's activity and his approach to interacting with people in Japan left the most impact on me because he demonstrated that community-centered projects can create a strong bond with the people you are surveying, but in order to do so, you must focus on listening to their stories and what they hope to achieve out of them. His presentation was especially memorable because the earthquake that happened in Japan in 2011 is very personal to me, and I remember a lot about it even though I was young. What I like to main from this class going forward is that when I work on my cultural geography research next year and start surveying people about their experiences and stories, I will focus on listening to them and treat them as real voices rather than just subjects of my research.

**Reply by albertkun** [reply]
@daikis12 To clarify, the third question is actually about connections to people in the class and/or visiting voices. But good to hear about how you will apply the class to your cultural geography research!

**Reply by daikis12** [reply]
Oops sorry I misunderstood the question...
I'd like to maintain the friendship that I've built with the people in the class, especially the brilliant (and wild) STEM girls in my group :)

**Reply by albertkun** [reply]
Haha, no problem! Good to hear about your group mates!

**Reply by XiongHer** [reply]
I also enjoyed learning and listening to Yoh's story and his work with local Japanese communities after the 2011 earthquake, the amount of effort, time, and resources that he put into the project and to build lifelong trust/connections with the communities. This is the kind of data justice approach and stories that we need more, to listen and represent locals' voices and concerns and to help them rebuild a community. Also, it was nice to hear how you will continue using this class work and knowledge in your other classes and in the future.

**Reply by shubhgupta2018** [reply]
I definitely resonate with you discussing the impact that Yoh's work had, as it was a really good reminder that community centered work is not easy (whether logistically or emotionally) but is really important and spiritually fulfilling as milestones are met. I would also love to learn more about your cultural geography project that you plan to take on next year!

**Comment by XiongHer** [comment]
Web development is deeply rooted in civil society, and it has great impacts, negatively or positively, depending on how and what we used web development for. Moving forward, there needs to be more awareness and practices of using ethics in technologies, continuing to use data and design justice approaches to listen and represent marginalized communities’ voices and issues. I enjoyed the last visiting voice from the Philippines (forgot his name). It was nice to hear about his works with Bernie Sander and other political campaigns, applying mapplication to solve real life issues and to advance certain social and political causes. Technology/mapplication will have great influences in the upcoming midterm elections. My group has been supportive and collaborative, so I just want to continue maintaining that friendship after this class is over. Also, we shared similar academic interests, so I guess we will collaborate again in future.

**Reply by daikis12** [reply]
I also thought that Rapi's presentation was very interesting and informative, and I liked it when he demonstrated that what we are learning in this class can be applied to solve real-life problems or start activism or movement for the greater good. After his presentation, I thought that shared ideas bring people together, and mapplication or good technology in general helps facilitate the interaction and communication among them. It's also good to hear that your group has been supportive and collaborative! I am looking forward to your presentation next week.

**Comment by shubhgupta2018** [comment]
_(Fair warning, I’m pretty emotional as I write this because it’s starting to hit me that I’m graduating soon but if you end up reading all of this I genuinely appreciate you uwu)_

As I reflect on my time at UCLA before turning my tassel in two weeks, I recognize the way that throughout my undergraduate experience, I treated technologies and tools such as web development as a crucial solution to societal problems that, although important, would not be my specialty. Instead, I saw my theory of change through grassroots movements and community organizing as more in tune with my personal goals. 

That said, Asian Am 191A has truly changed my perception of technology and its relation with social change. I’ve realized that while community organizing that directly address the material conditions of marginalized communities are important, web development and other tools of data collection, analysis, presentation, and visualization are fundamental to determine what specific material conditions should be changed in the first place. As Rapi Castillo’s experience in creating projects like Beautiful Rising demonstrate, community organizers can utilize technology to not only supplement but also **amplify** grassroots movements further. 

Of course, my mind then wonders to think about the gaps in CS departments and other technology heavy academic courses. As we have discussed in this course numerous times, the lack of emphasis of ethics in such areas of study unfortunately create a pipeline of students entering corporations that serve technocrats or perpetuate the military-industrial complex (this is not a critique of the students themselves and more of a criticism of the structures around these students that fail to share problematic weaponization of technology). However, our class’s visiting voice Mariah Tso played a major role in addressing my concerns with this question and therefore left the greatest impact on me. Her constant centering of indigenous communities and marginalized narratives within her lecture on what cartographers must do and know to be critical of structural oppression reminded me of this class’s theme that technology is not intrinsically bad but rather can be used against marginalized communities. In a similar sense, folks complicit in neoliberal politics and other forms of oppression can create technologies that are meant to be harmful (I.e. predictive policing).

Because I took this course P/NP so I could focus on grad school interviews and applications, I definitely did not put as much effort into building relationships with other students in this course as I wanted to. However, I am extremely grateful for Ashley and Haven for being some of the kindest and most compassionate individuals I’ve ever met. Ashley is ever so understanding, and I aspire to be as good at Javascript as she is. As for Haven, I’m going to miss her Pepe gifs and constant words of affirmation whenever I was going through it this quarter. I do hope that I can maintain such friendships even beyond my time at UCLA. And of course, huge shoutout to Professor Albert for being so kind and quite honestly one of the best professors I’ve ever had at UCLA!

So as I leave Los Angeles and move to Silicon Valley, I know that semicolons in Javascript will continue to be the bane of my existence. But at least I know that technology can help transform my radical love for my community into action that will materially change marginalized groups I care for through centering and amplifying their voices.

**Reply by NishuKrish** [reply]
I like how you brought up the role of ethics and technology and how many students at UCLA, especially those in STEM fields, are not exposed to the negative impacts that their work can cause. As you brought up, this has its own impacts, many end up working for companies that are destroying the most marginalized in our life. Professor Albert did an amazing job dismantling the narrative that technology is not the end-all, but merely one tool that can be used in our struggle for a just society. 
It was so nice to meet in you in class, Shubham!

**Reply by codingdianat** [reply]
Wow really thoughtful response! I am so happy that you had such a positive experience in this class (i think many of us had)! I wish you luck in your grad school interview and applications. With what you learned in this class, I am sure that you will make a strong impact wherever you go. I too had my perspective changed from this class. As someone who plans on working in tech, this class made me really think about the implications of tech development.

**Reply by ntran27** [reply]
This was so beautiful to read! I wish you the best with your transition to the Bay Area and hope that you'll be able to find a community within the industry that support the values we learned about in class. Congrats on graduation!

**Reply by angelineeow** [reply]
I feel you so much and getting emotional when writing this thinking cap. I love your response, it gets me so touched and not feel alone about being sad about leaving this school so soon. Congrats on graduating and I wish you luck in the future!

**Comment by NishuKrish** [comment]
I feel like this class left a big impact in me (Thank you so much, Professor Albert) as I realized to the extent in which web development and other digital tools are increasingly being used to harm communities. Big corporations and government are using data to target the most marginalized in our society. Growing up, I was always told about how technology was the "end all" and that it could magically solve all problems. However, many solutions do not take into account the perspective of the community and instead impose upon something that could make their lives worse.  This class taught me to **listen**, rather than just create solutions.  I feel like web development can be used as an important tool in our fight for a just and civil society as we head into the future, but only if communities are listened to. I found Machiko Yasuda's work with Maptime LA to be an important example of how such digital projects can be structured in order to center the community - having it be open source and listening to the community on their needs. Going forward, I am so happy to have found such an amazing community of fellow maptivists. Working with Ivy in our project (I would like to thank her for all of her amazing help and insights as we worked on our group project!) and everyone else in in-class discussions made me feel like I was connected to something larger than just myself. I especially would like to thank Professor Albert for teaching this class!; I feel like such a class is needed for anyone pursuing anything technical related (or really any field) as it is important to learn about the ethical uses (and limitations) of technology for social change.

**Reply by antonyblin** [reply]
I'm glad that you felt like this class taught you to listen and think before creating; you summed it up in words much better than I ever could have. Our multiple lectures on and practices of ethics really drilled into my mind the responsibility that comes with creation and representation. I am so glad that you had a great experience in this class, and I wish you the best with your future endeavors!

**Comment by codingdianat** [comment]
I think that the future of web development for civil society will continue to expand as web development becomes a more accessible skill. With more people equipped with web development knowledge, I believe that a new method of activism wil be utilized. Whether it is creating a map based website like our class or creating web application for politicians like Rapi, the impact of web development for civil society seems endless. This is why I believe that classes like Asia Am 191A are so important. So far, most of the tech classes that I have taken so far do not really delve into any social implications of the software that we learn. We only learn of its possibilities for innovation and all the “cool” things it could do. This class has reminded me that in order to create real change we must prioritize society. Although all the visiting voices were amazing in their own way, Rapi’s visiting voice really impacted me. This is because he basically called me out. He said, “As engineers we love coming up with solutions. What we lack is understanding the problem”. This really made me reflective of how I perceive any sort of design work that I do. I realized that I often think of what would be interesting to have without considering its impact. I am now trying to shift out of the previous mindset where I blindly design into one where I am more aware of its implications. I obviously still have a lot of work to do as Albert pointed out in the last lecture and through our office hours, but I am hopeful that I can improve. Moving forward from this class, I still hope to keep in contact with our amazing professor Albert! I feel like I learned so much in this class, mostly because of our captain! I hope I can ask for help in the future LOL. Also special thanks to my group project member Steven who has worked really hard on the project and has been a good team member!

**Reply by ashley-lu** [reply]
It's really cool to hear how this class had an impact on your perspective on design! As someone who works with data relatively frequently, I think I'm in the same boat in terms of learning how to keep civil society in mind when creating technology. As a more personal note, even though I've never had the chance to interact that much with you, I really appreciate your insight Diana 😄 You always ask such great questions to our visiting voices and I hope I can get a chance to talk to you more because you're an awesome person!

**Reply by ananyakodali17** [reply]
I thought that Rapi's statement about focusing more on the actual problem rather than the solution was a great way to shift your mind as well, and I think that it is great that you are working towards shifting your mindset even though it may be quite difficult. I also hope to work towards shifting my mindset in the same way to help and hear people rather than solve problems blindly, and this class really opened my eyes to that!

**Comment by ashley-lu** [comment]
I distinctly remember that on the first day of class, Albert asked us for examples of maps. At the time, none of us probably knew what we were getting into, but throughout this journey abroad Albert's pirate ship, we've grown from students aspiring to sail the seas of web development to full-fledged pirates who have gained knowledge of technology, civil society, and how they are intrinsically tied together. Now, when I look at a map, I no longer only see markers representing locations, but I also see the stories behind the people at those locations.

Through this class, I've developed my personal ethical views and have come to realize that web development is never a solution, but rather a platform for breaking down power imbalances and giving marginalized communities a voice. Creating technology that really manages to understand the communities it affects is much harder than we realize; if anything, the class project is a good example of that. We've run into so many challenges along the way, and we've revised the memo and survey more times than I can remember. Leading up to the final presentation, there's still so much we need to improve and fix. Yet I will say that this has been one of the most valuable experiences I've had at UCLA. I've learned so much through the class and project, and I hope I can carry my newfound knowledge with me as I continue towards the future.

The visiting voice that resonated the most with me was Nina Kin's collaboration and mentorship presentation. Though I've set myself up to take classes that I _think_ I would be interested in, I still sometimes feel lost in terms of what I want to do in the future. For me, it was reassuring to see that despite traversing through rough waters, Nina took the time to explore things she was interested in and ended up finding a place for herself in the intersection of government and technology. I think it's okay for me to stop continually pressuring myself to figure out my life and have fun discovering new experiences. 

Another part of her presentation I thought was important was focusing on building good relationships with other people. I'd like to think that this quarter, because of this class and because of personal growth, I was able to come out of my shell a little and make meaningful connections. First, I have to shoutout my groupmates Shubham and Haven for going through thick and thin with me throughout the course of this project. Shubham has been balancing classes with a mountain of grad apps, but they never fail to offer a great perspective on our project topic regarding PrEP for UCLA LGBTQ+ students. Haven is someone who I actually knew of previously from a shared club, but I had never actually interacted with her. Imagine my shock when I realized we were in the same club and we both played the cursed game Genshin Impact 🥲 Also would like to mention that I think I got to know Nish a bit better through this class, even though we knew each other already, and through my very few interactions with Lindsey in the Discord server, I would like to say she is a very cool person and she has a heavenly voice! I know I'll keep in touch with some of the people in our class, but I'd like to think that in the future, I can still maintain connections with others who I haven't interacted with as much.

Last but not least, this class wouldn't be the same without our captain Albert. I don't think I've ever met such an engaging lecturer and overall helpful and understanding professor. Listening to the aa191-rA-dio in vc together is one of my fondest memories of this class. If he ever decides to teach another class at UCLA for some reason, sign me up lol. Even if that's not the case, I hope I can continue to have great conversations with Albert in the future 😄

**Reply by willa-21** [reply]
Thanks for sharing your experience, Ashley! I think it's awesome that your team has grown closer! Your project topic is really important and it's great to see how this class has made a big impact on you. I also appreciated how Nina shared her career path and how she landed in government work.

**Comment by willa-21** [comment]
The future of web development for civil society has potential, but like any other professional field, there is a long way to go to truly change and impact society in a meaningful and positive way. “Location puts people at the center of what we do” - Rapi Castillo. I think Rapi’s presentation was really powerful. He demonstrated that maps and GIS can have real political change. The app/map called REACH shows that technology can bring people together, and the fact that AOC used it for her campaign is a good demonstration of this. I appreciate that he acknowledged that engineers “lack the understanding of the problem”, though I don’t think it has to be that way. Moving forward I also want to keep in mind the words of wisdom from all the speakers! I especially appreciated how Tiffany Guerra-Huang brought concepts of mindfulness into her design and justice work. Thanks to Albert for bringing in some great speakers who are doing amazing work. I also enjoyed working with my group on the class project and definitely plan to stay in touch with them!

**Reply by ananyakodali17** [reply]
I agree that web development for civil society has potential but there is a lot of work to be done first. All of the speakers gave us such great information and knowledge, but I really enjoyed Tiffany's as well!

**Reply by emmashanson** [reply]
I agree that Rapi's work is powerful and his words are inspiring! Seeing his connection staring from Github and being contacted by the Sander's team to use his map was an interesting story to hear.

**Comment by ananyakodali17** [comment]
I think that I honestly learned the most useful material in these past 10 weeks through this class than any other classes. We learned about everything from the basics of HTML, JavaScript, and CSS to the ethics of data collection and presentation, and most importantly how to empower communities and focus on the why? of a problem rather than just finding a solution. Web development currently doesn't focus on the consumer and the marginalized communities that may be harmed, and I think that this class really helped me realize the ties between society and technology and how we can only fix the issues of society using technology, and vice versa. I think that web development has a long way to go to become something that truly benefits civil society, but the visiting voices helped me see that there are people out there making those steps. I thought that Tiffany's presentation left the most impact on me, because it was incredible how she used design justice to make such moving statements. Thank you Albert for this amazing class! I hope to stay in contact with my group partner after this class.

**Reply by ntran27** [reply]
I enjoyed Tiffany's presentation as well. I thought her newspaper design comparing the multiple pandemics of anti-Black racism and covid-19 were really powerful in showing the human impact of both.

**Comment by ntran27** [comment]
Technologies for web development will likely continue to develop in functionality and expand concerning their potential impact on civil society.  When I think of places that have regular and consistent access to what we think of as advanced technological innovations, I think there will also be the push and pull of how such technologies are being used. With web development, we are consistently seeing how the internet and websites are used as a platform to disseminate both empirical information and misinformation. I think as more people potentially gain skills in web development, there is a great possibility that those skills will be used to contribute toward a common good in civil society and/or used to isolate and hide extremist ideologies/communities which can harm society. 

My favorite visiting voice was Tiffany. I appreciated the resources she shared with us, including the images and art she created. It spurred my interest in developing the skills to create similar artwork as a way to visualize the impacts of social inequities to more creatively communicate. I also liked Yoh’s presentation and hearing about his experiences moving all around the world and finding a career he seems to be passionate about. It is a dream of mine to be able to find that fulfillment in work while achieving personal goals such as traveling and seeing the world and having an impact on society. 

Regarding connections, I enjoyed working with my group and do plan to stay in touch with them!

**Reply by kris10fu** [reply]
I agree that more people are becoming interested in web development and this increasing widespread use can have both advantages and disadvantages. As a result, I think learning about the ethics regarding technology is more important than ever and should be required. I also really enjoyed Tiffany's presentation and thought that the designs she shared with us showed how to utilize creative visualizations for civil society and community engagement.

**Comment by emmashanson** [comment]
The most impactful visiting voice that I've met would be Nina given that I had the closest sense of relatability with her in terms of background and where she came from and her path to what she does now.  I really liked her presentation and how she connected to a TV show that she really likes and hearing about her goals for her own life as well as the impact on the community she wants to place is important.  As another women in STEM, seeing her journey was inspiring and I hope to stumble upon my niche that I am proud to be a part of post grad.
As for personal connections I want to keep, I will say that while in the beginning of the class, I was confused as to how my project would impact the community, but after working on it more and more with Kristen and Keri, I have a better understanding of it and I would like to personally continue work with the Little Tokyo community more in the future. I have in the past, but my experience has been limited so I would like to improve those connections more!
As for the future of web development, I can foreshadow larger companies continue to try to take user information for things like targeted ads, but on the other hand, with ideally greater public knowledge of how our data being used is very harmful, the general public would be more inclined to hold these corporations accountable and demand to see change and transparency. In large tech companies, they used to reply on lack of knowledge to exploit their user in an non-exploitive way (very long terms and conditions page which disclose all of this information but the user is not very inclined to actually read it) but with organizations breaking down key takeaways of the information being used, the public has improved and will continue to improve their understanding of data usage.

**Reply by stefuntran** [reply]
Nina's presentation was one of the best in my opinion because I could feel her passion for the actual topic discussion and her passion for the show. I'm glad you are able to find her journey inspiring and relatable!

**Reply by penelopelamb** [reply]
I feel the same way about Nina's presentation. Her relatability definitely kept me engaged and I admired her authenticity.  Hearing her journey was insightful, too. Plus, the TV show connection gave me a new show to watch!

**Comment by stefuntran** [comment]
Prior to taking this class, my impression of web development for civil society was always thinking about how these websites are providing resources and solutions to their target audience. I did not think about design in terms of hearing the input of what the user or community has to say. I believe the future of web development for civil society is going towards an upward trend because more people are learning about this concept through workshops and classes just like this one! The visiting voice that left the most impact on me was Yoh Kawano because of his commitment to the communities he has helped. I resonated with him when he discussed how he would get to know the people of Fukushima instead of just extracting data from them because that is something that I hope to do when I have the chance to do field research and interviews. It is so critical to listen to the people and what they have to say as it humanizes the research experience and reminds people that the goal of this research is to help these communities at the end of the day. My group partner, Diana, has done a tremendous job throughout the assignments this quarter despite it being just the two of us. I hope to stay in touch with her!

**Reply by kericsp** [reply]
I really like that you mention the changing perception from solutions-oriented to community-oriented! I also went through a similar reframing of mind over the course of this class, and I think Yoh did a really good job of demonstrating the value of commitment and follow-through to the impacted communities!

**Reply by joonyloony** [reply]
i agree with your sentiment about Yoh Kawano!  I really enjoyed his presentation and it was super engaging to see the various people he interviewed and their backgrounds. His interview with the high school girl was especially powerful to me, since it gives us a look into the lives of regular (and young) people who were living through the nuclear radiation/fallout in the area.

**Comment by kris10fu** [comment]
The future of web development for civil society will incorporate more data justice and design justice principles and guidelines. These two movements can implement more discussion and activity around ethics in technology, particularly for companies or businesses as that is something that is seriously lacking. Furthermore, using web development to empower and uplift marginalized, overlooked communities is another important direction for the future. Encouraging individuals from such communities to actively participate in the process or supporting technological products made by these communities would help increase equity in terms of technology and web development. I also think there will be a lot more innovation when it comes to the web in the future, and it will be interesting to see developments in this area. The visiting voice that left the most impact on me would be Yoh’s presentation. I was inspired by both his backstory and the projects that he’s done with local communities. I related to his background of moving to different countries and thought that his journey to figuring out his career was really interesting. Hearing about his story with the Fukushima prefecture provided me with valuable insight on how to center the community’s stories as the focus rather than outside perspectives. I think that hearing about the strong personal connections he created with the people there also showed the importance of consistency when it comes to long-term community engagement projects to properly respect the community. I would like to maintain personal connections with my group members, Emma and Keri. I think that we worked well together, and it was fun to work on our project. The technical skills in web development that I obtained were also incredibly helpful for future projects. I learned a lot from this class, and it challenged me to think more about my personal values and ethics when it comes to community engagement and civil society. Thank you to Albert for teaching and guiding us this quarter, and I hope to stay in touch with him as well!

**Reply by hajovel** [reply]
I really liked your point that participation of these communities is important to help increase equity and allyship, as I also agree. The platform for these communities to speak themselves is super important and I appreciate you bringing it up. I also resonated most with Yoh's story as well, and also found his connections and long-term engagement with these people inspiring!

**Comment by kericsp** [comment]
Web development has made strides in the private sector and especially in the world of big tech, but I feel that we still have miles to go before web development for civil society catches up. However, I feel like this isn't necessarily a bad thing-- where big tech companies have been living by mottos like "move fast and break things" and "don't be evil" (oh the irony...), civil society and activism is driven by grassroots efforts and really takes into account the effects of digital tools on society and people over profit. This class has been truly eye-opening for me. I came to this class from a place of moderate experience with coding, web dev, and digital tools and a passion for creating a better world but with little knowledge on how to do something meaningful with my life and my abilities without ending up as some cog in a corporate machine. 

That being said, Tiffany's visiting voice definitely had the biggest impact on me. I really appreciate that she shared her story of struggling to find her place and going through a "quarter-life crisis", because I could definitely relate! I actually followed up with her after her visiting voice presentation and asked if she had any career advice for me and shared some details about myself, and to my surprise she actually wrote me back! 

Going forward, I hope to keep in touch with my classmates from this course -- Emma Hanson and I are actually officers in the same club, NSU. As part of NSU, we went on a Little Tokyo walking tour that eventually inspired our project on Little Tokyo Towers/Miyako Gardens! I hope to make more connections between what I'm learning in my classes as part of the Digital Humanities minor (which this class is an elective for), Asian American studies, and the community. Above all I really value how Prof. Kochaphum has instilled values and ethics into the way this class is taught, showing understanding and compassion for students despite teaching a typically strict, "hard science" topic (coding). I'm grateful that I took this class!

**Reply by hajovel** [reply]
I also have a similar concern of ending up, as you said, another cog in the machine. I share a similar sentiment, coming with some technical experience but not really any civil experience. I also resonated with Tiffany's story and am glad she reached back - I hope with whatever she shared with you, its useful and leads you the right direction~!

**Comment by hajovel** [comment]
My thoughts about the future of web development for civil society is that there is still a long way to go before ethics in technology really becomes a foundational idea when creating technology. However, due to classes like this, I do not believe that technology will tend towards being inherently bad by the misusage of large companies, as more people become aware of the power it holds. I hope that with classes like these, developers and designers who visited to speak in this course (and of course, Albert :)) that spread their knowledge as well, that ethics in technology becomes fundamental in web development.

For visiting voices, I found Yoh’s story and work to be the most impactful. To be honest, I am very pessimistic towards researchers, people with privilege and power, entering or visiting marginalized or affected communities of tragedies as oftentimes, unfortunately, it is for their own gain and sensationalism. However, the way he spoke about who he kept connections with, felt very genuine. Hearing the impact those people left on him, his perception of the world, and the impact his work had not on him but the people affected, really inspired me. I was left very inspired, and want to connect and serve communities similarly to Yoh and the people of Fukushima. 

As for personal connections, I am very grateful I got put into a group with Ash and Shubham, people who are very talented and motivate me to be a better version of myself. They’re both a pleasure to get to know, and I hope after this class, Ash and I stay connected with our mutual interests and club, and Shubham knows I’m rooting for him wherever he goes! I would also like  to say taking this class has definitely changed my perception on so many things. This was one of the most engaging classes I’ve had and I’ve definitely grown as a person. I now know the intersection between ethics and technology and the dire need for it and being critical of tech and web, and it’s all thanks to this course, thank you Albert! I hope I can stay in touch and hear more of your puns!

**Reply by bittertang** [reply]
In a way the future of web development felt reassuring because of the developers and designers that came to speak :) You make a really good point about the sad reality of researcher intentions and it was also really inspiring for me the way Yoh was able to make lasting connections with the community.

**Comment by antonyblin** [comment]
There's a lot of work to be done for the future of web development for civil society. It's great that web development is becoming ever more accessible outside of formal academics and technical schooling through programs like W3Schools, MDN, FCC, bootcamps, and programs like them, but the mass production of capable programmers trained in the basics has made for an influx of programmers and coders entering the famed "tech industry" with very little idea of the impact their creations and innovations can have beyond product development. This is obviously understandable for the purposes of the individual making a living and supporting oneself and family, but there needs to be more of an emphasis on ethics and classes like this one to better educate budding programmers.

I think my absolute favorite visiting voice was one we had at the beginning of this course: Mariah. Not only was she a really outstanding speaker and presenter, but she was able to effectively and beautifully demonstrate her knowledge, findings, heritage, and arguments. I appreciated most how she connected her Diné heritage and other indigenous cultures to critical cartography and the significance of properly representing the community being studied and mapped.

I've had a really difficult time due to some personal issues, and I really appreciate the support of Kaitlyn and Professor Kochaphum, and I would really like to keep in contact with them after this class. I've also met some amazing and inspirational individuals like Shubham in class, and they are also people that I would like to continue to connect with after this course is over.

**Reply by kaittli** [reply]
Your point on how individuals disregard ethics because they are focused on making a living is exemplified a lot in society. There seems to be a lot of emphasis on hard technical skills or digital tools but not much importance placed on soft skills, which is the main thing that is lacking in technological spheres. I agree that with the growth of  online resources committed towards enhancing technical skills, the incorporation of ethics is essential. Hope to finish the class off strong!

**Comment by bittertang** [comment]
Although this might be very optimistic, I have hope in the future of web development for civil society! I think that with classes such as Asian Am 191a and just increasingly accessible educational resources in general, more and more attention and efforts are being dedicated to using technology descriptively as opposed to prescriptively. Ethics are slowly but surely becoming more integrated in tech culture, and even in design, for example, the knowledge of designing accessibly is becoming standard for designers. Out of all the brilliant visiting voices we had, the one that resonated with me the most was Yoh because of his own personal involvement with the community. The fact that he’s still able to maintain the relationships he made with the community members even after he left really went to show his dedication to the project. It really inspired me, and I hope to be able to have that same passion towards a project, and to be able to work with individuals and community members at that level one day. 

Thanks to this class, I was able to get a better grasp on HTML, CSS, and (especially) Javascript and being able to now possess the skillset to create a mapplication that will uplift a community has been such a great learning experience. I’ve gained so much insight about how technology can be used as a platform to listen and represent experiences in a way that is empowering. As for the community we’ve built in our class, I’d love to maintain connections with everyone! I found it really interesting how our community, despite coming from various majors and background, all share a desire to learn how to incorporate technology in efforts towards social good. Special thank you to Nish for working with me in creating a mapplication for educational inequity, a cause we’re both passionate about! And lastly, thank you to Albert for all the work he does, especially the weekly awards and puns! We’re all super appreciative of the time and effort you put into this class :)

**Reply by spolcharoen** [reply]
I also really found that Yoh left an impact on me when he came to speak! His ability to show up consistently to interact with community members even after his main project is "over" is so cool. I also like your optimistic outlook on the future of web dev for civil society. I think we have to have some of that optimism to make progress! :)

**Comment by penelopelamb** [comment]
As society becomes increasingly reliant on technology, I think it's important to leverage web development as a platform for social activism. In fact, after taking this class, I'd argue that a good portion of the future of activism will leverage technology. I joined this class after an enrollment mixup by recommendation of my roommate and project partner, Sine, and it was a great experience overall. Out of all of the visiting voices, I related the most to Nina (yay women in EE) and found her perspective around her career to be insightful. Whether she likes it or not our "personal connection" will be maintained as my roommate throughout next year, and as my host for when we go to Thailand next summer.

**Reply by spolcharoen** [reply]
Mi casa es su casa <3

**Comment by spolcharoen** [comment]
As we approach the end of this source, my thoughts around the future of web development for civil society have definitely shifted. Beforehand, I didn't really know the full extent of what data and design justice could be, but knowing now that there are so many great minds working towards it gives me some optimism. Although there is still a marathon to go in terms of fully integrating ethics and activism into tech, I believe courses such as these are a good sign that at least we are headed in some correct direction. 

The visiting voice that left the most impact on me was Yoh. Having taken his course before, I really enjoyed listening to him talk about his work once again and uncovering details I didn't remember from before. He particularly left an impact on me because of his experience moving around and feeling not tied to one specific homeland. I related to this and was inspired by the work he has done.

I would like to maintain connections with my slayer groupmate and roommate Lillan, as well as connections with Professor Albert and the rest of the class. Compared to my other STEM classes, I really enjoyed how receptive this class was to discussion and help requests. Thank you for making all those amazing slideshows. It was fun making those BTS maps for y'all. <3

**Comment by joonyloony** [comment]
I think that the future for web development for civil society has a lot of potential. There are so many corrupting factors that could potentially lead it down the incorrect path, but it has been very inspiring to see so many people from different backgrounds actively fighting against those capitalist and corrupt influences in order to work towards a solution. I think for myself though, my journey with web design may end here (lol). I personally took this class since I saw it advertised on Reddit the summer prior and I didn’t get the opportunity to take it, and I’m so glad I had the opportunity to do so this quarter. I’ve learned a lot regarding coding and especially css <3 but I honestly do not see myself continuing doing web design and GIS in the future for enjoyment or as a career. 

I think that Yoh Kawano’s presentation left the biggest impact on me, and I really enjoyed seeing his journey through the world and the community engagement that he displayed when working in Japan. The diverse collection of people he surveyed gave very powerful testimonies and it was really moving to see that they were happy to see him come back time and time again, which really shows how important having a good relationship with your community you are surveying is. I had actually already attended a presentation by him when I was a fall quarter sophomore and newly learning about (so over a year ago) and it’s great to see how his work has evolved over a year and I’m happy that he is able to move back to Japan!

I would hope to stay in touch with my group member who has been a very dedicated collaborator in the past 10 weeks, and I really appreciate her commitment and reminding me that certain assignments are due. I would also hope to keep in touch with Albert since he has provided a lot of help and guidance to my outside coding projects and I think that he is an invaluable connection :)

**Comment by kaittli** [comment]
Reflecting on my time in this class, I am hopeful that ethics will become more incorporated into web development. Technology’s vast influence means that it has the potential to reach people of different backgrounds, classes, and creeds, all with the ability to empower communities. Because technology is becoming increasingly intertwined with society, it is imperative that individuals can learn how to do so—through learning principles in data/design justice, opensource, technology ethics, etc, the future of technology may shift towards connecting communities together. Throughout the course, I have learned how empowerment can be exemplified in simply listening to other people’s perspectives, reflecting on what things matter, or thinking about the purpose of what we are creating; through community efforts, I am optimistic that people will begin to emphasize these values.

While all the visiting voices were amazing, I was most touched by Yoh Kawano’s backstory with helping communities with disaster relief, specifically after the nuclear disaster in Fukushima. He showed true care to those communities by maintaining connections with the people impacted by the disaster over the years, ensuring that those people and their stories are not being forgotten. It helped emphasize to me that although sometimes there is nothing that can be “fixed”, as in this case, giving people an outlet to share their sentiments and experiences is equally valuable. Listening to other people's stories can stimulate conversation to work towards positive impacts.

Shoutout to my groupmate Antony for our late nights of working, the lame jokes we crack, the bad ways we enable one another, and our understanding of each other's circumstances. Also, thank you so much Professor for being such a supportive instructor through the rough patches of not being able to keep up with VSCode as well as for doing such a great job at integrating ethics along with technology. I am super thankful of all the extra effort into ensuring that we as students were comfortable with the course content as well as the puns to make class extra enjoyable and the personalized feedback on our work. Thank you so much for teaching this class and enlightening us with all of your knowledge!

**Comment by angelineeow** [comment]
I think that web development for civil society should never be about solutions to a problem. Therefore, I think the future of web development for civil society should always be listening to the communities who needed help and wish to be heard. In Albert's class, I learned that web development is about empowering the communities who needed our help and never just giving solutions. Those who use web development for civil society should always have empathy and try to be in the community's shoes when they are trying to help.

The visiting voice that left the most impact on me is Machiko Yasuda. Her experience of transitioning into mapping which is a whole different career path from what she learned during college inspired me so much since I am graduating soon. It might be unbelievable but her stories of figuring out her life and how she is involved with the community in various ways got me motivated in pursuing my interest in social change. I am very thankful and honored to be able to listen to her stories and other visitors' stories as they all inspired me in different ways and eventually come down to one important point, listen to those who need our help.

I would like to thank my groupmates, Lindsey and Daiki. They helped me so much and I really enjoy being with them outside of class. (Extremely thankful for grouping us together) I would also like to thank our professor, Albert! Thank you so much for helping me throughout the quarter and really enjoy learning in your class. I love the content of the class and think this is the most enjoyable and useful class. Words can't express the feelings I have, I am extremely grateful to meet every one of you in this class. :)
